client/src/pages/Login.js

```diff
@@ -114,7 +114,7 @@ function Login(props: Props) {
               <div className="white-text">Remember me</div>
             </Checkbox>,
           )}
-          <a className="login-form-forgot" href="home">
+          <a className="login-form-forgot" href="password-reset">
```

Comment:
Instead of an <a> tag can we make this a <Link>? <a> tags refresh the page, where <Link>s don't

Reply:
Sure, do you want me to change that for all the other places where an <a> tag is used as well?

Reply:
Ahh yes thatd be great!

Reply:
Well turns out there was only one other place it was used it was the place I referenced it from